data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
 			`	"When your name came up in the study," he says, "I asked around and discovered that you have helped us in other ways as well. The other prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall henceforth be given permission to purchase our ships and technology."`
 			choice
 				`	"I'm honored. Thank you."`
-					decline
+					goto onwards
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
-				decline
+			`	"I do not have anything at the moment," he says, "but I am sure other Remnant will have tasks you can help with, now that you have demonstrated your trustworthiness. We have been keeping track of you, and we are impressed with your curiosity, perseverance, and ability to survive where most would be destroyed. All three are valued characteristics among the Remnant, and thus many will welcome your assistance on their projects.`
+			label onwards
+			`	As he talks, he gestures for you to walk with him towards a nearby hall. "While you are welcome to become one of us, we suspect that you are a citizen of the galaxy - wandering hither and yon building coalitions and enmities with species and groups across the breadth of accessible space, unfettered by conventional allegiances. As such, we hope that you will continue to explore and stop by to let us know what you have found. We ask only that you keep the secret with which you are trusted, and bring us word of anything that might influence humanity's long-term survival."`
```

Comment:
This is a really powerful bit of dialogue. I feel like it would be a good point for the player to be given the opportunity to respond to it in various ways. The next `choice` would IMO be more impactful if it were moved up to be right after this bit of dialogue, with the bit about going to the cafeteria moved earlier. And then you could add a few more choices, such as (paraphrasing):
- "Actually I really like Remnant space."
- "Yes, though I've enjoyed my time here, I can never stay in one place for long."

Reply:
Thanks! Currently working on implementing this.

Reply:
Implemented.